build/nsm/docker/Test.Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1 as build
+ENV PACKAGEPATH=github.com/ligato/networkservicemesh/
+RUN apk add --update protobuf git bash
```

Comment:
The golang base doesn't have apk, thus causing travis to be sad and fail. I believe the golang base is Debian based and you may want a "apt-get install protobuf git bash" here. Note I may have the package names wrong, but you can figure those out ;).